### DISCUSSION TITLE
Crop indexed color PNG and keep the same palette in output

### DISCUSSION THREAD

**Discussion by pladaria** [ImageMagick/ImageMagick#3719] [Help]
I'm unable to find a way to crop an indexed color PNG and keep the same palette in output

```
$ convert --version
Version: ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org
Copyright: © 1999-2019 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC Modules OpenMP 
Delegates (built-in): bzlib djvu fftw fontconfig freetype jbig jng jpeg lcms lqr ltdl lzma openexr pangocairo png tiff webp wmf x xml zlib
```

Is that possible?

**Comment by snibgo** [comment]
Have you tried "-define png:preserve-colormap=true"? See https://imagemagick.org/script/defines.php.

**Reply by pladaria** [reply]
yes, without luck

I also opened a stackoverflow question about this
https://stackoverflow.com/questions/67635441/crop-an-indexed-png-with-imagemagick-and-keep-the-same-palette-as-the-input-file

It is kinda funny but my current workaround is to use ffmpeg:
```
ffmpeg -i $INPUT -vf "crop=8:32:0:0" $OUTPUT
```

**Reply by HorstBaerbel** [reply]
Also see #416

**Comment by Jmuccigr** [comment]
Have you tried v. 7? Could you post a sample image?

**Reply by HorstBaerbel** [reply]
Try converting multiple different image with "+remap" to get the same color palette for all of them.
